kernel/cClosure.ml

```diff
@@ -127,13 +127,17 @@ type evar_handler = {
   evar_expand : constr pexistential -> constr evar_expansion;
   evar_repack : Evar.t * constr list -> constr;
   evar_irrelevant : constr pexistential -> bool;
+  qnorm : Sorts.QVar.t -> Sorts.Quality.t;
   qvar_irrelevant : Sorts.QVar.t -> bool;
```

Comment:
seems like qvar_irrelevant is redundant with qnorm (`qvar_irrelevant q = Quality.equal (qnorm q) Quality.irrelevant`)
not sure if we care